### ISSUE TITLE
unweighted(n()) on "tbl_svy" objects throws rlang warning

### ISSUE THREAD

**Issue by mlaviolet** [gergness/srvyr#54]
When calling `unweighted(n(())` on a `tbl_svy` object, I get the warning noted below. Apparently the triple-bang is no longer to be used to unquote a single quosure. I've encountered this warning on my own functions, but wasn't able to find a solution. Thanks!

```{r}
library(srvyr)
data(api, package = "survey")
dclus1 <- as_survey_design(apiclus1, id = dnum, weights = pw, fpc = fpc)
dclus1 %>% 
  group_by(sch.wide) %>% 
  summarize(n = unweighted(n()))

# Warning message:
#   Unquoting language objects with `!!!` is deprecated as of rlang 0.4.0.
# Please use `!!` instead.
# # Bad:
# dplyr::select(data, !!!enquo(x))
# 
# # Good:
# dplyr::select(data, !!enquo(x))    # Unquote single quosure
# dplyr::select(data, !!!enquos(x))  # Splice list of quosures`
```

**Comment by bschneidr** [comment]
It looks like the simplest solution in this particular case is just to switch to using "!!" instead of "!!!" within `unweighted()`, which should be ok since the `x` argument is mean to be a single variable name or a single expression.

**Comment by gergness** [comment]
Awesome, thanks for reporting @mlaviolet and for the quick fix @bschneidr . Please do keep reporting these deprecation warnings from rlang, they are hard to keep up with!

**Comment by gergness** [comment]
closed via #55
